src/websocket-client.ts

```diff
@@ -124,7 +125,7 @@ interface ListenKeyPersistenceState {
   keepAliveFailures: number;
 }
 
-function throwUnhandledSwitch(x: never, msg: string): never {
```

Comment:
Please revert this line, it's designed to throw a build error if a switch statement no longer covers all possible values in a union type (e.g. if a new WsMarket is added but not yet handled by the switch statement).